R/retentionIndexing.R

```diff
@@ -52,6 +52,58 @@ indexRtime <- function(x,
   
 }
 
+#' @title 2-point correction of RIs
+#'
+#' @description
+#'
+#' `correctRindex` performs correction of RIs based on reference substances
+#'
+#' @param x `numeric` vector with retention indices
+#' 
+#' @param y `data.frame`data.frame containing two columns, where the first 
+#'     holds the measured RIs of the reference substances and the second the
+#'    reference RIs
+#'
+#' @return `numeric` vector of same length as x with correction retention 
+#'     indices. Values are floating point decimals. If integer values shall be
+#'      used conversion has to be performed manually
+#'
+#' @author Michael Witting
+#'
+#' @export
+#'
+#' @examples
+#' 
+#' ref <- data.frame(rindex = c(110, 210),
```

Comment:
Maybe also here add a little more description what will be done and what the individual data sets are?